Makefile

```diff
@@ -144,29 +146,21 @@ NMSSH: force fatlibssh2
 
 libgit2: force
 	mkdir -p submodules/libgit2/build
-	MAKE=/usr/local/bin/cmake PATH=/usr/local/bin:${PATH} cd submodules/libgit2/build && ${CMAKE} -DBUILD_SHARED_LIBS=OFF -DCMAKE_OSX_ARCHITECTURES="x86_64;arm64" -DCMAKE_OSX_DEPLOYMENT_TARGET="10.14" -DCMAKE_INSTALL_PREFIX=../../../ThirdParty/libgit2 ..
-	MAKE=/usr/local/bin/cmake PATH=/usr/local/bin:${PATH} cd submodules/libgit2/build && ${CMAKE} -j22 --build . && ${CMAKE} --build . -j22 --target install
-	MAKE=/usr/local/bin/cmake PATH=/usr/local/bin:${PATH} cd submodules/libgit2/build && ${CMAKE} --build . -j22 --target install
-
-deps: force
-	make fatlibsixel
-	make fatopenssl
-	make fatlibssh2
-	make CoreParse
-	make NMSSH
-	make bindeps
-	make libgit2
-       
+	PATH=/usr/local/bin:${PATH} cd submodules/libgit2/build && ${CMAKE} -DBUILD_SHARED_LIBS=OFF -DCMAKE_OSX_ARCHITECTURES="x86_64;arm64" -DCMAKE_OSX_DEPLOYMENT_TARGET="10.14" -DCMAKE_INSTALL_PREFIX=../../../ThirdParty/libgit2 ..
+	PATH=/usr/local/bin:${PATH} cd submodules/libgit2/build && ${CMAKE} --build . --target install --parallel "$(sysctl -n hw.ncpu)"
```

Comment:
This needs double dollar signs `$$(sysctl -n hw.ncpu)` or else make evaluates it to empty string.